testsuite/features/build_validation/init_clients/ubuntu2004_minion.feature

```diff
@@ -12,7 +12,7 @@ Feature: Bootstrap a Ubuntu 20.04 Salt minion
 
   Scenario: Bootstrap a Ubuntu 20.04 minion
     Given I am authorized as "admin" with password "admin"
```

Comment:
Do we still need this here? Why not use the new scenario with the login?

Reply:
same for all the rest really

Reply:
Fixed.